tests/Integration/TestCase.php

```diff
@@ -193,4 +277,250 @@ public function go_to_new_post( string $post_status = 'publish' ): int {
 	public function set_admin_user( $admin_user_id = 1 ): void {
 		wp_set_current_user( $admin_user_id );
 	}
+
+	/**
+	 * Assert availability of hooks.
+	 *
+	 * @param string[] $hooks WordPress hooks whose availability we have to verify.
+	 * @param bool     $availability_type TRUE if we want to check the presence of given hooks.
+	 *
+	 * @return void
+	 */
+	public function assert_wp_hooks_availablility( $hooks, $availability_type ) {
+		if ( ! $this->isPHPVersion7Dot2OrHigher() ) {
```

Comment:
Due to some reason `has_action` isn't working PHP 7.1 so using this function in codebase to avoid the checks.

I could have used just one approach i.e. calling the function directly instead of using `do_action` but chose this approach because `do_action` is more close to how WordPress works i.e. until we support PHP 7.1 we will use both approaches to write better tests and to make PHP 7.1 happy.

Reply:
Maybe we would like to add a PHP comment about this, in order to not lose the context in the future?

Reply:
Yes, the function name is self-explanatory. I was referring more to the "why we need to do this". But feel free to mark this as resolved without any changes, it is not an issue.